db/api/api.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const { handle500, handle404 } = require("../api/controllers/handleErrors");
+const { getCategories } = require("./controllers/index");
+const app = express();
+
+app.get("/api", (req, res) => {
```

Comment:
Im not sure you need this path...yet. I presume the ticket hasn't asked for it. 
I'd make sure to only include the code needed for the ticket.